### ISSUE TITLE
Uncaught (in promise) ReferenceError: __ is not defined

### ISSUE THREAD

**Issue by Pondake** [statikbe/laravel-nova-chained-translation-manager#17]
Whenever you want to edit a string.

![translationbug](https://user-images.githubusercontent.com/4643209/198969183-4b51b271-2e2e-41a9-8d83-34046d1ed500.gif)

**Comment by 7ammer** [comment]
It looks like localisation has not been imported.

Snippet from nova docs: https://nova.laravel.com/docs/4.0/customization/localization.html#frontend
```
<template>
  <h2>{{ __('Total Users') }}</h2>
</template>

<script>
import { Localization } from 'laravel-nova'

export default {
  mixins: [Localization]

  // ...
}
</script>
```

**Comment by enricv** [comment]
Same problem, but the translation is saved.

**Comment by gamladenov** [comment]
Same problem ... translation is saved.

**Comment by stijnelskens** [comment]
@enricv @gamladenov @7ammer @Pondake 
This has been fixed in the latest release v3.0.4
